generators/templates/src/index.coffee

```diff
@@ -17,16 +17,21 @@ catch error
   logger.error error
   
 
-# Index page
+# Index page displaying info about the service. The optional req query param is just for example purposes
 app.get('/', (req, res) ->
-  res.status(200).send("#{info.name} #{info.version}")
+	res.status(200).send("#{info.name} #{info.version} #{if req.query.name? then req.query.name else ' '}")
 )
 
 # Swagger page
 app.get('/+swagger', (req, res) ->
   res.sendFile('swagger.yaml', {root: './static/'})
 )
 
+# echo POST endpoint, just to have an example of reusing params at swagger file
+app.post('/+echo', (req, res) ->
+	res.status(200).send("Hi. #{if req.query.name? then req.query.name else 'provide a name as query param'}")  
```

Comment:
Could you instead add this at the top:
```
# Utility
ensureParameters = (req, res, parameters...) ->
  for p in parameters
    if !req.query[p]?
      res.status(400).send("Parameter #{p} required")
      return false
  true
```

then the function content becomes like this:
```
 if ensureParameters(req, res, 'name')
   res.send(req.query.name)
```
This way we have the ensureParam util also in this template

Reply:
👍